docs/references/test-client.md

```diff
@@ -0,0 +1,373 @@
+    ...
+    2022-10-17T18:57:02.826512Z DEBUG plugin_execution: ayx_sdk_cli::commands::plugin::run::tool_service: Attempting to acquire extension runtime tool service port assignment...
+    2022-10-17T18:57:03.602284Z  INFO plugin_execution: ayx_sdk_cli::commands::plugin::run::tool_service: Acquired port from runtime tool service tool_service_port=53241
+    2022-10-17T18:57:03.602301Z  INFO plugin_execution: tool_client: Connecting to extension runtime tool service tool_service_address="http://localhost:53241"
+
+
+## Commands
+
+### Completion
+
+#### list
+
+Show the available list of shells for which the test client can provide
+an auto-completion script.
+
+    ayx-sdk-cli.exe completion list
+    Bash
+    Zsh
+    PowerShell
+    Elvish
+    ...
+
+#### show
+
+Provide the auto-completion script for a given shell.
+
+Step 1: Create a new completion file.
+
+`ayx-sdk-cli.exe completion show powershell > $profile/ayx-sdk-cli-completions.ps1`
+
+Step 2: Add the following to the end of `$profile`.
+
+`. $PSScriptRoot\ayx-sdk-cli-completions.ps1`
+
+### Extension
+
+#### list
+
+List extensions found in the current workspace.
+
+    ayx-sdk-cli.exe extension list
+    ...
+    Found 9 extension(s):
+    - ConversionPassthrough v1.0.0 (runtime target: ayx-sdk-extension-python-3_8_5)
+    - Input v2.0.0 (runtime target: ayx-sdk-extension-python-3_8_5)
+    - Optional v2.0.0 (runtime target: ayx-sdk-extension-python-3_8_5)
+    - Passthrough v2.0.0 (runtime target: ayx-sdk-extension-python-3_8_5)
+    - MultipleInputs v2.0.0 (runtime target: ayx-sdk-extension-python-3_8_5)
+    - DcmInput v1.0.0 (runtime target: ayx-sdk-extension-python-3_8_5)
+    - MultiConnection v2.0.0 (runtime target: ayx-sdk-extension-python-3_8_5)
+    - MultipleOutputs v2.0.0 (runtime target: ayx-sdk-extension-python-3_8_5)
+    - Output v2.0.0 (runtime target: ayx-sdk-extension-python-3_8_5)
+
+
+#### extension
+
+Provide information about a specific extension.
+
+    ayx-sdk-cli.exe extension info Passthrough
+    ...
+    Name           : Passthrough
+    Package Name   : N/A
+    Description    : An example single-input-single-output tool. This tool takes data from the input anchor and, without changing it, writes it to the output anchor.
+    Kind           : com.alteryx.sdk.tool
+    Version        : 2.0.0
+    Runtime Target : ayx-sdk-extension-python-3_8_5
+    Category       : Python SDK Examples
+    Copyright      : 2022
+
+
+### Plugin
+
+#### run
+
+Runs a given set of inputs against an SDK plugin. Shows any messages
+from the plugin in stdout, including responses. Writes any outputs to
+the specified outputs given.
+
+`ayx-sdk-cli.exe plugin run [WORKSPACE_PATH]::[PLUGIN_NAME] --input <[SOURCE][:TYPE[:OPTIONS]][@ANCHOR]> --output <[DESTINATION][:TYPE[:OPTIONS]][@ANCHOR]> --runtime-command`
+
+For more information, go to [Test
+Client](https://help.alteryx.com/developer-help/test-client).
```

Comment:
Point this to the new reference on ayx-developer-sdk repo.

This should point to the corresponding markdown version